mod.ts

```diff
@@ -5,7 +5,7 @@ export {
   number,
   // Already exported by command module.
   // ValidationError,
-  OptionType,
+  type OptionType,
```

Comment:
thats not correct, an enum is a value and not a type. but i think i made the same mistake in `flags/mod.ts`.
```suggestion
  OptionType,
```

Reply:
Are you fixing the export in flags/mod.ts as well? or should i create a new PR?